docs/content/en/docs/community/contributing.md

```diff
@@ -5,80 +5,129 @@ description: >
+involve the community on the discussion.
+
+### GitHub Discussions
+
+GitHub Discussions are used for feature requests (that don't have actionable items/issues), questions, and anything else
+the community would like to share.
+
+Categories:
+* Q/A - Questions
+* Proposals - Feature requests and other suggestions
+* Show and tell - Anything that the community would like to share
+* General - Everything else (possibly announcements as well)
+
+### GitHub Issues
+
+GitHub Issues are used to file bugs, work items, and feature requests with actionable items/issues (Please refer to the
+"Reporting Bugs/Feature Requests" section below for more information).
+
+Labels:
+* "\<area\>" - area of project that issue is related to (create, upgrade, flux, test, etc.)
+* "priority/p\<n\>" - priority of task based on following numbers
+  * p0: need to do right away
+  * p1: don't have a set time but need to do
+  * p2: not currently being tracked (backlog)
+* "status/\<status\>" - status of the issue (notstarted, implementation, etc.)
+* "kind/\<kind\>" - type of issue (bug, feature, enhancement, etc.)
+
+Refer to the [template](https://github.com/aws/eks-anywhere/tree/main/.github/ISSUE_TEMPLATE) for more information on
+what goes into an issue description.
+
+### GitHub Milestones
+
+GitHub Milestones are used to plan work that is currently being tracked.
+
+* next: changes for next release
+* next+1: won't make next release but the following
+* techdebt: used to keep track of techdebt items, separate ongoing effort from release action items
+* oncall: used to keep track of issues needing active follow-up
+* backlog: items that don't have a home in the others
+
+### GitHub Projects (or tasks within a GitHub Issue)
+
+GitHub Projects are used to keep track of bigger features that are made up of a collection of issues.
+Certain features can also have a tracking issue that contains a checklist of tasks that
+link to other issues.
 
 ## Reporting Bugs/Feature Requests
 
-We welcome you to use the GitHub issue tracker to report bugs or suggest
-features.
+We welcome you to use the GitHub issue tracker to report bugs or suggest features that have actionable items/issues
+(as opposed to introducing a feature request on GitHub Discussions).
 
-When filing an issue, please check
-[existing open](https://github.com/aws/eks-anywhere/issues) or
-[recently closed](https://github.com/aws/eks-anywhere/issues?q=is%3Aissue+is%3Aclosed),
-issues to make sure somebody else hasn't already reported the issue. Please
-try to include as much information as you can. Details like these are
-incredibly useful:
+When filing an issue, please check existing open, or recently closed, issues to make sure somebody else hasn't already
+reported the issue. Please try to include as much information as you can. Details like these are incredibly useful:
 
 * A reproducible test case or series of steps
-* The version of the code being used
+* The version of our code being used
```

Comment:
Is there a benefit of changing this? I've talked about it in the past and prefer "the" because as the community contributes features the word "our" has connotations that this isn't a community project

Reply:
I just missed this